model/registry/user-agent.yaml

```diff
@@ -10,3 +10,18 @@ groups:
           Value of the [HTTP User-Agent](https://www.rfc-editor.org/rfc/rfc9110.html#field.user-agent) header sent by the client.
         examples: ['CERN-LineMode/2.15 libwww/2.17b3',
                    'Mozilla/5.0 (iPhone; CPU iPhone OS 14_7_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.2 Mobile/15E148 Safari/604.1']
+      - id: device.name
+        type: string
+        brief: >
+          Name of the device extracted from original
+        examples: ['iPhone']
+      - id: name
+        type: string
+        brief: >
+          Name of the user-agent extracted from original. Usually refers to the browser's name
```

Comment:
How about linking this as a reference?
https://www.whatsmyua.info/?q=Mozilla/5.0%20(iPhone;%20CPU%20iPhone%20OS%2012_1%20like%20Mac%20OS%20X)%20AppleWebKit/605.1.15%20(KHTML,%20like%20Gecko)%20Version/12.0%20Mobile/15E148%20Safari/604.1